--- lapack-3.4.2/BLAS/SRC/CMakeLists.txt.sover~	2010-08-10 18:57:20.000000000 +0200
+++ lapack-3.4.2/BLAS/SRC/CMakeLists.txt	2013-02-13 03:42:25.385943586 +0100
@@ -141,4 +141,8 @@ if(UNIX)
   target_link_libraries(blas m)
 endif()
 target_link_libraries(blas)
+set_target_properties(blas PROPERTIES
+    VERSION ${LAPACK_VERSION}
+    SOVERSION ${CPACK_PACKAGE_VERSION_MAJOR}
+)
 lapack_install_library(blas)
--- lapack-3.4.2/CMakeLists.txt.sover~	2013-02-13 03:42:25.385943586 +0100
+++ lapack-3.4.2/CMakeLists.txt	2013-02-13 03:46:41.686836962 +0100
@@ -1,5 +1,7 @@
 cmake_minimum_required(VERSION 2.8)
 project(LAPACK Fortran)
+set(LAPACK_VERSION 3.4.2)
+set(CPACK_PACKAGE_VERSION_MAJOR 3)
 
 # Configure the warning and code coverage suppression file
 configure_file( 
@@ -216,8 +218,6 @@ endif(LAPACKE)
 SET(CPACK_PACKAGE_NAME "LAPACK")
 SET(CPACK_PACKAGE_VENDOR "University of Tennessee, Univ. of California Berkeley, Univ. of Colorado Denver and NAG Ltd")
 SET(CPACK_PACKAGE_DESCRIPTION_SUMMARY "LAPACK- Linear Algebra Package")
-set(LAPACK_VERSION 3.4.2)
-set(CPACK_PACKAGE_VERSION_MAJOR 3)
 set(CPACK_PACKAGE_VERSION_MINOR 4)
 set(CPACK_PACKAGE_VERSION_PATCH 2)
 set(CPACK_RESOURCE_FILE_LICENSE "${CMAKE_CURRENT_SOURCE_DIR}/LICENSE")
--- lapack-3.4.2/SRC/CMakeLists.txt.sover~	2013-02-13 03:42:25.385943586 +0100
+++ lapack-3.4.2/SRC/CMakeLists.txt	2013-02-13 03:44:42.637506806 +0100
@@ -416,4 +416,8 @@ list(REMOVE_DUPLICATES ALLOBJ)
 
 add_library(lapack ${ALLOBJ} ${ALLXOBJ})
 target_link_libraries(lapack ${BLAS_LIBRARIES} ${XBLAS_LIBRARY})
+set_target_properties(lapack PROPERTIES
+  VERSION ${LAPACK_VERSION}
+  SOVERSION ${CPACK_PACKAGE_VERSION_MAJOR}
+)
 lapack_install_library(lapack)
