--- CMakeLists.txt.lib64	2011-05-18 11:00:23.000000000 +0200
+++ CMakeLists.txt	2011-05-18 11:02:36.000000000 +0200
@@ -166,7 +166,7 @@
 install(FILES
   ${LAPACK_BINARY_DIR}/CMakeFiles/lapack-config.cmake
   ${LAPACK_BINARY_DIR}/lapack-config-version.cmake
-  DESTINATION lib/cmake/lapack-${LAPACK_VERSION}
+  DESTINATION lib${LIB_SUFFIX}/cmake/lapack-${LAPACK_VERSION}
   )
 install(EXPORT lapack-targets
-  DESTINATION lib/cmake/lapack-${LAPACK_VERSION})
+  DESTINATION lib${LIB_SUFFIX}/cmake/lapack-${LAPACK_VERSION})
